parsec/scheduling.c

```diff
@@ -559,7 +560,9 @@ int __parsec_context_wait( parsec_execution_stream_t* es )
         if( task != NULL ) {
             misses_in_a_row = 0;  /* reset the misses counter */
 
+            PARSEC_PINS(es, SELECT_END, task);
```

Comment:
not clear why we need to move these calls from inside to outside of this function.